Lib/fontTools/feaLib/builder.py

```diff
@@ -764,12 +764,28 @@ def buildGDEFGlyphClassDef_(self):
             classes = {g: c for (g, (c, _)) in self.glyphClassDefs_.items()}
         else:
             classes = {}
+            ignoreClassFlagsUsed = False
             for lookup in self.lookups_:
+                ignoreClassFlagsUsed = (
+                    ignoreClassFlagsUsed or (
+                        lookup.lookupflag & (
+                            otl.LOOKUP_FLAG_IGNORE_BASE_GLYPHS +
+                            otl.LOOKUP_FLAG_IGNORE_LIGATURES +
+                            otl.LOOKUP_FLAG_IGNORE_MARKS
```

Comment:
minor: you could union these three flags before the for loop in some local (or even global) variable instead of doing it for each lookup, result is always the same